### ISSUE TITLE
“.NET MAUI gallery” not announced as heading.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6594]
GitHubTags:
#.NET MAUI; #AILimited; #A11ySev2; #A11yMAS; #A11yTCS; #Android; #AndroidPhone; #WCAG1.3.1; #GH_.NETMAUI_Android_Apr2022; #Headings​;

### Description

“.NET MAUI gallery” is not defined as a heading.

Similar issue observed in the following scenarios:

.NET MAUI<<Recipes Search<<Search Box: “Search all recipes” not announced as heading.
.NET MAUI<<My Recipes: “Recipes” not announced as heading.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Swipe till “.NET MAUI gallery” and observe the issue.

https://user-images.githubusercontent.com/86710788/165717540-0f89018a-0ff1-4b30-af3d-a94cca9d8fc8.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
“.NET MAUI gallery” should be given Heading Level 1.
```

**Comment by rachelkang** [comment]
Hi, @SuryacharanM - could you help us understand when a label should be marked as a heading vs. not?
We noticed that the text in the title bar doesn't get announced as a heading on the Android platform (i.e. the "Settings" title of the system settings screen doesn't get announced as a heading) and were wondering how this situation differs

**Comment by SuryacharanM** [comment]
Github tags: #CEACCApproved; #A11yCAIOH; #Closed;
As per the discussion, closing the bug.
